src/chrome/content/rules/Bitmask.net.xml

```diff
@@ -18,9 +18,15 @@
 	<!--	Direct rewrites:
 				-->
 	<target host="bitmask.net" />
-	<target host="demo.bitmask.net" />
-	<target host="dl.bitmask.net" />
+	<target host="*.bitmask.net" />
 
```

Comment:
deb wasn't targeted before this change, so you're introducing a problem and fixing it.

Would a better description for your intention be "cover all but deb in *.bitmask.net"? If so, are you reasonably certain there aren't any other problematic hosts?